### DISCUSSION TITLE
Hurricane Force Wind Warning not returning data during event

### DISCUSSION THREAD

**Discussion by formula4** [weather-gov/api#521] [Q&A] [answered]
Greetings.

There is currently an active warning for zone ANZ800 for a hurricane force wind warning, but there is no JSON data being returned by the API.  (1/30/22, 8:00Z).  Below is the warning found on a forecast.weather.gov WWA summary page :

**Offshore Waters Forecast
NWS Ocean Prediction Center Washington DC
1016 PM EST Sat Jan 29 2022

ANZ800-301430-
Gulf of Maine to the Hague Line-
1016 PM EST Sat Jan 29 2022**

I also see this active on the WWA display at https://www.spc.noaa.gov/products/wwa/.

I know that the far offshore zones are not supported by the API yet, but ANZ800 looks to be in the inner two ocean zones where WWAs are served by the API. I also know ANZ800 is listed in the available zones on the API. 

Does the fact that the WWA was issued by the ocean prediction center vs. a WFO have an effect on it being delivered for the API to distribute?  Or is this a bug? I am also thinking that it's possible the API is not returning any Hurricane Force Wind Warnings at all, as I bumped into this issue in 2020 with the Hazardous Weather Outlook and Short Term Forecast, but after an email to NCO (INC0243794) the issue was resolved. 

Just a side note/question - is there a list of known WWAs that don't return data? I am not assuming the issue is that widespread, but some of these 129 are very obscure and who knows how long it might be until it's noticed, even by myself... and I have been working with them daily.

**Comment by jah1007** [comment, marked as answer]
The High Seas Zones (those managed by the Ocean Prediction Center) are not provided in CAP format by the upstream data provider. There is currently no requirement now or future, at the moment, to make them available. Hazardous Weather Outlooks and Short Term forecasts are deemed forecast information and not alert information so they are purposefully not made into CAP format.

**Reply by formula4** [reply]
Thank you for going over that, this will make it easier for me to know what to expect in the future. Just an idea - would it be possible to remove the WWAs on the list located at https://api.weather.gov/alerts/types that don't return data? I see the Hazardous Weather Outlook and Short Term Forecasts listed there, but they obviously from what you said do not return anything. For now, I am assuming those are the only ones that are like that. Thanks again for the insight.

**Reply by jah1007** [reply]
That would be a question for @StephenClouse.
